pkg/tablebackup/tablebackup.go

```diff
@@ -27,10 +27,10 @@ import (
 
 const (
 	// TODO: move those constants somewhere else
-	DeltasDirName              = "deltas"
-	BasebackupFilename         = "basebackup.copy"
-	BaseBackupStateFileName    = "info.yaml"
-	BackupStateFileNamePattern = "backup_state_%d.yaml"
+	DeltasDirName           = "deltas"
+	BasebackupFilename      = "basebackup.copy"
+	BaseBackupStateFileName = "info.yaml"
+	DeltasState             = "deltas_state.yaml"
```

Comment:
I think it is better to use explicit OIDs in the file name instead of relying on them to be in the correct directory. We may also consider explicitly writing table OIDs to the actual files; that would save us in case of a confusion (i.e. a filesystem glitch resulting in some files coming to lost + found or someone accidentally copying the file to the wrong place and overwriting an existing one from another table).